src/main/java/nl/techiteasy/controller/Exceptions/RecordNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package nl.techiteasy.controller.Exceptions;
+
+public class RecordNotFoundException extends RuntimeException {
+    private static final long serialVersionUID = 1;
```

Comment:
Deze regel is in principe niet nodig, weet je waar het voor dient? Anders is het misschien leuk om eens op te googlen :)